db/migrate/20231202063417_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :text
+      t.integer :comments_counter
+      t.integer :likes_counter
```

Comment:
- [ ] Please adhere 🙏🏻 to the project requirements, you did great work but I think you forgot the establish a foreign `relation` between the `user` and `post` table by using the `alias` column name as `author_id` in the `post` table. Kindly consider the things carefully and implement the changes as I just mentioned and take a look at the below screenshot.


![image](https://github.com/lucy-sees/Blog/assets/66476329/9311b22a-7b3f-4870-a599-89818279d6f5)

### Instruction
![image](https://github.com/lucy-sees/Blog/assets/66476329/3be36b1d-dced-4c06-b3f2-9117af8cd8b9)